apps/src/lab2/levelEditors/levelData/EditMusicToolbox.tsx

```diff
@@ -129,6 +134,16 @@ const EditMusicToolbox: React.FunctionComponent<EditMusicToolboxProps> = ({
             blockMode={blockMode}
             toolboxType={toolbox?.type}
           />
+          {toolbox?.type === 'flyout' && (
+            <Checkbox
+              checked={!!addFunctionCalls}
+              name="addFunctionCalls"
+              label="Add function calls"
+              onChange={event => {
+                onAddFunctionCallsChange(event.target.checked);
+              }}
+            />
+          )}
```

Comment:
Knowing that the K5 module is also in development, could we add a note to the UI to explain that this option should currently only be used if all function definitions are set to uneditable/undeletable in start sources?

I also wonder if we might want to explain why the call blocks don't show in the preview workspace (because they're dynamically created based on the real student workspace).

Reply:
Screenshots and code updated, per our follow-up discussion.  Thanks!